src/test/functional/shared/apiHelpers.ts

```diff
@@ -18,12 +19,34 @@ export const createUserWithRoles = async (email, password, forename, userRoles)
       {
         headers: {'Content-Type': 'application/json'},
       })).data;
-
   } catch (e) {
     throw new Error(`Failed to create user ${email} with roles ${userRoles}, http-status: ${e.response?.status}`);
   }
 };
 
+export const createUserWithSsoId = async (email, password, forename, userRoles,ssoId) => {
```

Comment:
This method here is virtually identical to the one before with an additional ssoId field. We can use the same method and check for whether ssoId is undefined, and return the additional field for ssoId in the json of that it present. Read stack overflow here for how to do it https://stackoverflow.com/questions/456177/function-overloading-in-javascript-best-practices :)